apps/favorite_products/priv/static/js/phoenix.js

```diff
@@ -0,0 +1,1026 @@
+(function(exports){
```

Comment:
Newer versions of Phoenix have this file pulled directly from deps, preventing you from needing to check it in. See the upgrade guide: https://gist.github.com/chrismccord/d5bc5f8e38c8f76cad33